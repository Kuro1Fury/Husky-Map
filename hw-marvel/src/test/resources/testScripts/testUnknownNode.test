# Tests unknown nodes can be detected

CreateGraph g
AddNode g n1

FindPath g n1 n2

FindPath g n3 n2