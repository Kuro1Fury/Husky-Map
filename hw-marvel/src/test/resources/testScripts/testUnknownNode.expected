# Tests unknown nodes can be detected

created graph g
added node n1 to g

unknown: n2

unknown: n3
unknown: n2